venza.py

```diff
@@ -6,6 +6,32 @@
 HS_URL = "http://hladnystudent.zones.sk/jedalne-listky-{}-{}-{}"
 FF_URL = "http://www.freefood.sk/menu/"
 
+MEAL_TYPE_SOUP = 0
+MEAL_TYPE_MAIN_DISH = 1
+MEAL_TYPE_OTHER = 2
+
+
+class Meal(object):
+    name = ''
```

Comment:
@pepol well, yeah, they no longer make any sense here, given we set these in `__init__`.